lib/install/utils.go

```diff
@@ -62,6 +63,7 @@ func GetAppPackage(service app.Applications) (*loc.Locator, error) {
 func GetApp(service app.Applications) (*app.Application, error) {
 	apps, err := service.ListApps(app.ListAppsRequest{
 		Repository: defaults.SystemAccountOrg,
+		Type:       storage.AppUser,
```

Comment:
This ensures that we don't use an unrelated application package if the user application is actually missing from the tarball.